README.md

```diff
@@ -149,16 +148,15 @@ configured by setting the `manager` option to `subscription_manager`:
 
 ```ruby
 Vagrant.configure('2') do |config|
-...
+  ...
   if Vagrant.has_plugin?('vagrant-registration')
     config.registration.manager = 'subscription_manager'
   end
-...
+  ...
 end
 ```
 
-In case of `subscription_manager` manager for the preferred registration pair,
-you would be ask on your username/password combination.
+In case of you have choosen `subscription_manager` as manager, you would be asked for your username/password combination.
 
```

Comment:
In case you choose `subscription_manager` as the manager, you would be asked for your user credentials, such as the username and password.